src/components/CartComponents/cartNav/CartNav.jsx

```diff
@@ -1,9 +1,24 @@
-import { Link } from "react-router-dom";
+import { Link, useNavigate } from "react-router-dom";
 import { useCart } from "../../context/cart-context";
+import { useAuth } from "../../context/auth-context";
```

Comment:
one suggestion You can use an ```index.js ``` file and add all your exports here and you can simply import all your contexts from a single file.